pkg/controller/stats/aggregator.go

```diff
@@ -340,6 +381,36 @@ func (a *Aggregator) doCollect(summary *controlplane.NodeStatsSummary) {
 			a.networkPolicyStats.Update(curStats)
 		}
 	}
+	if features.DefaultFeatureGate.Enabled(features.Multicast) {
+		reportedGroups := sets.NewString()
+		for _, mcastGroupInfo := range summary.Multicast {
+			group := mcastGroupInfo.Group
+			reportedGroups.Insert(group)
+			a.groupNodePodsMapMutex.Lock()
```

Comment:
acquring lock is not free, seems not worth to lock and unlock repeatedly here to exclude a few code from critical area, which may take more cost

Reply:
updated